### ISSUE TITLE
run.py changes

### ISSUE THREAD

**Issue by apatel726** [apatel726/HurricaneDissertation#15]
1. Need to add code to include making directories for models on dates created
- i.e mkdir for current date inside models folder to save if running new model
2. ability of code to append epochs and dropout to models

**Comment by apatel726** [comment]
add a log for start and stop times for model run

**Comment by hammad93** [comment]
Need to add code to include making directories for models on dates created

This can be done by also create some metadata for these models.

**Comment by hammad93** [comment]
It's recommended to use the SavedModel approach since it also saves the model architecture so there's no guesswork
https://www.tensorflow.org/tutorials/keras/save_and_load#savedmodel_format

**Comment by hammad93** [comment]
Please note that the new file names are appended with an ISO 8601 timestamp as a unique identifier

**Comment by hammad93** [comment]
Some required changes include,
- adding a timer to run.py
- adding a naming convention, `u` for univariate and `m` for multivaritate

**Comment by apatel726** [comment]
need to finalize change in run.py for singular run case logs

log(pprint.PrettyPrinter(indent=4).pprint(bidir_lstm_model_wind.model.get_config()))
-.model needs to be added to lines 91, 97, 102 before get_config()))

**Comment by hammad93** [comment]
The hyperparameter search should be based on the following article, 

https://machinelearningmastery.com/grid-search-hyperparameters-deep-learning-models-python-keras/

**Comment by apatel726** [comment]
separete keras metric link: https://www.tensorflow.org/api_docs/python/tf/keras/metrics/MeanAbsolutePercentageError

**Comment by hammad93** [comment]
add training time indicator to logs and save them to the directory.

this should be the last change before closing the commit.
